frontend/src/config/config.ts

```diff
@@ -0,0 +1 @@
+export const serverURL = "http://localhost:5000";
```

Comment:
the server url must be environment variable.

Reply:
Good catch! fixed it.